src/components/MessageActions/MessageActions.js

```diff
@@ -123,8 +124,11 @@ const MessageActionsWrapper = (props) => {
     typeof customWrapperClass === 'string'
       ? customWrapperClass
       : defaultWrapperClass;
-  /** @type {() => void} Typescript syntax */
-  const onClickOptionsAction = () => setActionsBoxOpen(true);
+  /** @type {(e: React.MouseEvent) => void} Typescript syntax */
+  const onClickOptionsAction = (e) => {
+    e.stopPropagation();
```

Comment:
react 17 caused this issue, unless we stop the propagation, the action box immediately closes